File: code_samples/java/TestingFixtures.java
/**
 * Testing: Fixtures
 * AI/ML Training Sample
 */
public class TestingFixtures {
    
    private String data;
    
    public TestingFixtures() {
        this.data = "";
    }
    
    public void process(String input) {
        this.data = input;
    }
    
    public String getData() {
        return this.data;
    }
    
    public boolean validate() {
        return this.data != null && !this.data.isEmpty();
    }
    
    public static void main(String[] args) {
        TestingFixtures instance = new TestingFixtures();
        instance.process("example");
        System.out.println("Data: " + instance.getData());
        System.out.println("Valid: " + instance.validate());
    }
}

File: code_samples/swift/testing_fixtures.swift
import Foundation

/// Testing: Fixtures
/// AI/ML Training Sample

class Fixtures {
    var data: String
    
    init() {
        self.data = ""
    }
    
    func process(_ input: String) {
        self.data = input
    }
    
    func getData() -> String {
        return self.data
    }
    
    func validate() -> Bool {
        return !self.data.isEmpty
    }
}

// Example usage
let instance = Fixtures()
instance.process("example")
print("Data: \(instance.getData())")
print("Valid: \(instance.validate())")
